sample/F8/sessions

```diff
@@ -0,0 +1,1249 @@
+        },
+        "comments": "", 
+        "language": null, 
+        "slides": null, 
+        "audio": null, 
+        "video": null, 
+        "signup_url": null, 
+        "state": "accepted", 
+        "timezone": "UTC",
+        "session_type": {}, 
+        "track": {},
+        "start_time": "2017-04-18T08:00:00.000Z",
+        "end_time": "2017-04-18T19:00:00.000Z"
+    }, {
+        "short_abstract": null,
+        "long_abstract": "null",
+        "id": 2,
+        "title": "Breakfast",
+        "subtitle": "day-1-breakfast",
+        "microlocation": {
+            "id": 2,
+            "name": "LL"
+        },
+         "comments": "", 
+        "language": null, 
+        "slides": null, 
+        "audio": null, 
+        "video": null, 
+        "signup_url": null, 
+        "state": "accepted", 
+        "timezone": "UTC",
+        "session_type": {}, 
+        "track": {},
+        "start_time": "2017-04-18T08:00:00.000Z",
+        "end_time": "2017-04-18T10:00:00.000Z"
+
+    }, {
+
+        "short_abstract": null,
+        "long_abstract": "null",
+        "id": 3,
+        "title": "Keynote",
+        "subtitle": "day-1-keynote",
+        "microlocation": {
+            "id": 3,
+            "name": "City National Civic"
+        },
+         "comments": "", 
+        "language": null, 
+        "slides": null, 
+        "audio": null, 
+        "video": null, 
+        "signup_url": null, 
+        "state": "accepted", 
+        "timezone": "UTC",
+        "session_type": {}, 
+        "track": {},
+        "start_time": "2017-04-18T10:00:00.000Z",
+        "end_time": "2017-04-18T11:30:00.000Z"
+
+    }, {
+
+        "short_abstract": null,
+        "long_abstract": "null",
+        "id": 4,
+        "title": "Lunch",
+        "subtitle": "day-1-lunch",
+        "microlocation": {
+            "id": 4,
+            "name": "Festival Hall"
+        },
+         "comments": "", 
+        "language": null, 
+        "slides": null, 
+        "audio": null, 
+        "video": null, 
+        "signup_url": null, 
+        "state": "accepted", 
+        "timezone": "UTC",
+        "session_type": {}, 
+        "track": {},
+        "start_time": "2017-04-18T11:30:00.000Z",
+        "end_time": "2017-04-18T12:30:00.000Z"
+
+    }, {
+        "short_abstract": null,
+        "long_abstract": "null",
+        "id": 5,
+        "subtitle": "introduction-to-the-camera-effects-platform",
+        "title": "Introduction to the Camera Effects Platform",
+
+        "sessionDescription": "Come hear more about Camera Effects Platform, a suite of creative tools that lets you create effects for the Facebook camera, from simple photo frames to interactive masks and effects that use Augmented Reality (AR). In this session you’ll learn what camera effects are, how to create and submit your own, and how you can connect with the audiences that matter to you by providing rich experiences that help them express themselves.",
```

Comment:
why create a new attribute when we have "short_abstract" and "long_abstract" for gathering the information of the event?